tests/vertexonly/test_vertex_only_mesh_generation.py

```diff
@@ -64,6 +64,11 @@ def parentmesh(request):
         m = UnitIcosahedralSphereMesh()
         m.init_cell_orientations(SpatialCoordinate(m))
         return m
+    elif request.param == "immersedsphereextruded":
```

Comment:
This needs to be added to the params above right?

Reply:
Fixed